maven-extension/src/main/java/io/opentelemetry/maven/OtelExecutionListener.java

```diff
@@ -66,6 +77,28 @@ public OtelExecutionListener() {
               + mojoGoalExecutionHandlers.entrySet().stream()
                   .map(entry -> entry.getKey().toString() + ": " + entry.getValue().toString())
                   .collect(Collectors.joining(", ")));
+
+      // help debugging class loader issues when the OTel APIs used in
+      // Maven plugin mojos are mistakenly not loaded by the OTel Maven extension
+      // causing the lack of context propagation from the OTel Maven extension to the plugin mojos
+      ContextStorage contextStorage = ContextStorage.get();
+      logger.debug(
+          "ContextStorage: "
+              + contextStorage
+              + ", identity="
+              + System.identityHashCode(contextStorage));
```

Comment:
Nit: consider using slf4j placeholders instead of concatenating strings:
```suggestion
      logger.debug(
          "ContextStorage: {} , identity={}", contextStorage, System.identityHashCode(contextStorage));
```

Reply:
Sure. Can you wrap it with `log.isDebugEnabled()` then?

Reply:
Sure -- I didn't notice that this entire block is actually wrapped in a `logger.isDebugEnabled()` condition.